src/data.jl

```diff
@@ -177,6 +181,16 @@ end
 nkpt(b::BandStructure{D}) where D = nkpt(b.kpts)
 nband(b::BandStructure{D}) where D = nband(b.bands[1])
 
+"""
+    FatBands{D} <: AbstractReciprocalSpaceData{D}
+
+Stores information useful in plotting FatBands.
+"""
```

Comment:
It might be good to update this docstring with more information on fat bands (though that doesn't have to be included in this particular pull request).